client/src/common/styledComponents/Button.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'vue-styled-components';
+
+const btnProps = { primary: Boolean };
+
+const StyledButton = styled('button', btnProps)`
```

Comment:
Do you think we need to prefix all our styled components with `Styled`? To me it looks a bit redundant

Reply:
Agree